src/witan/send/adroddiad/pptx/slides.clj

```diff
@@ -0,0 +1,162 @@
+(ns witan.send.adroddiad.pptx.slides
+  (:require [kixi.plugsocket :as kps]
+            [witan.send.adroddiad.slides :as was]))
+
+(defn bulleted-list [seq-of-text]
+  (->> seq-of-text
+       (map #(str "- " %))
+       (clojure.string/join "\n\n")))
+
+(defn chart-box [conf]
+  "expects a map for a vega-lite chart"
+  {:slide-fn :chart-box
+   :vega-lite-chart-map (:chart conf)
+   :width (cond
+            (contains? conf :left-col)
+            (/ (- 1920 100) 2)
+            :else
+            (- 1920 100))
```

Comment:
this calc is repeated in text-box and table-box and could probably be factored out into a function